src/component/resource/ResourceSummary.tsx

```diff
@@ -53,17 +52,11 @@ export class ResourceSummary<P extends ResourceSummaryProps = ResourceSummaryPro
         });
     };
 
-    protected onRemoveClick = () => {
-        this.setState({showRemoveDialog: true});
-    };
-
     public onRemove = () => {
-        this.props.removeResource(this.props.resource);
-        this.setState({showRemoveDialog: false});
-    };
-
-    protected onRemoveCancel = () => {
-        this.setState({showRemoveDialog: false});
+        const onCloseRemove = this.onCloseRemove;
```

Comment:
IMHO, it is not necessary to use a temporary variable for callback when using fat arrow function.